packages/core/src/shipping/consignment.ts

```diff
@@ -48,6 +48,7 @@ export interface ConsignmentUpdateRequestBody {
     shippingAddress?: AddressRequestBody;
     lineItems?: ConsignmentLineItem[];
     pickupOption?: ConsignmentPickupOption;
+    shippingOptionId?: string;
```

Comment:
Can this not be resolved in the backend? I don't think extending this interface is idea. This property exists in the consignment interface already. So should be able to pull it from quote object in backend.

Reply:
Fixed. Used another method `selectShippingOption`